webui/src/pages/repositories/index.jsx

```diff
@@ -206,21 +176,12 @@ const RepositoriesPage = () => {
                         </Form.Row>
                     </Form>
                     <ButtonToolbar className="justify-content-end mb-2">
-                        <Dropdown>
-                            <Dropdown.Toggle variant="success" id="template-picker-dropdown">
-                                <RepoIcon/> Create Repository
-                            </Dropdown.Toggle>
-                            <Dropdown.Menu>
-                                <Dropdown.Item onClick={() => {
-                                    setShowCreateRepositoryModal(true);
-                                    setCreateRepoError(null);
-                                }}>Blank Repository</Dropdown.Item>
-                                <Dropdown.Item onClick={() => {
-                                    setShowRepositoryTemplatesModal(true);
-                                    setCreateRepoError(null);
-                                }}>Spark Quickstart</Dropdown.Item>
-                            </Dropdown.Menu>
-                        </Dropdown>
+                        <CreateRepositoryButton variant={"success"} enabled={true} onClick={
```

Comment:
I'd suggest moving the onClick callback from an anonymous arrow function to a named function + `useCallback`